server/server.go

```diff
@@ -131,6 +131,7 @@ func Start() {
 
 	router := httprouter.New()
 	router.GET("/", indexHandler)
+	router.GET("/healthz", healthHandler)
```

Comment:
Why is this endpoint needed?

Reply:
It's being used in deployment's `readinessProbe`, [here](https://github.com/mattermost/matterbuild/pull/31/files#diff-45ed3b7096bcaff1633c99d50b30e99aR45-R48).